src/index.js

```diff
@@ -221,17 +218,15 @@ function bookRequestedTrip() {
     };
     let requestedTrip = fetcher.fetchTripRequest(int);
     Promise.all([requestedTrip])
-        .then(fetchStuff(+user.id))
-        .then(switchBackAfterTrip())
-        .then(alert(`Your trip to ${allDestinations.find(location => location.id === wantedTrip.destinationID).destination} has been booked!`))
-
+        .then(() => fetchStuff(+user.id))
```

Comment:
Good catch on using the anonymous function.  This is necessary every time, otherwise I believe the function just get's called back on page load.